config/more_samples-glyf_colr_1.py

```diff
@@ -1076,42 +1195,42 @@ def main():
     glyphs = [
         SampleGlyph(glyph_name=".notdef", accessor="", advance=600, glyph=Glyph()),
         SampleGlyph(glyph_name=".null", accessor="", advance=0, glyph=Glyph()),
-        _sample_sweep(-360, 0, "pad", "narrow", next(access_chars)),
+        _sample_sweep(-360, 0, "pad", "narrow", position, next(access_chars)),
```

Comment:
my understanding from https://github.com/googlefonts/colr-gradients-spec/pull/362 is that the proposed PaintSweepGradient's "biased" start/endAngles do not allow to represent angles < -180 degrees (nor >= 360+180==540), exactly because of the +1.0 bias

So this -360 nominal start angle one would blow up with a struct.error when compiling with a copy fonttools that accounts for the angle's +1.0 bias.

Similarly all the following `_sample_sweep` where the angles are < -180 or >= 540